content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,68 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updates to existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing resource - for example, a ConfigMap -
+using an HTTP PUT. For a PUT request, it is the client's responsibility to specify
+the `resourceVersion` (taking this from the object being updated). Kubernetes uses
+that `resourceVersion` information so that the API server can detect lost updates
+and reject requests made by a client that is out of date with the cluster.
+In the event that the resource has changed (the `resourceVersion` the clienti
+provided is stale), the API server returns a `409 Conflict` error response.
+
+Instead of sending a PUT request, the client can send an instruction to the API
+server to **patch** an existing resource. A **patch** is appropriate if the
+change that the client wants to make isn't conditional on the existing data.
```

Comment:
I don't think this (conditional or not) is the top of line consideration when requesting a patch.
A patch operation is mostly about "partial update", which means I don't want to **replace** an existing object. I'm interested in changing one or more attributes as the desired state.
Considering the "level-triggering" nature of k8s API, most of the time the server doesn't care about "lost updates". This is especially true for cases like scaling a replica set.

Reply:
> Considering the "level-triggering" nature of k8s API, most of the time the server doesn't care about "lost updates". This is especially true for cases like scaling a replica set.

I've definitely seen people suggest performing patches that could lose updates. Incrementing the scale of a ReplicaSet or Deployment? Probably fine. Updating a specific key within a ConfigMap that represents a _last modified_ timestamp? You can do it, but it's up the client to pick the right operation.